packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -747,30 +753,28 @@ export class ForkChoice implements IForkChoice {
     // is from a prior epoch to the attestation, then the target root must be equal to the root
     // of the block that is being attested to.
     const expectedTarget =
-      target.epoch > computeEpochAtSlot(block.slot)
-        ? indexedAttestation.data.beaconBlockRoot
-        : fromHexString(block.targetRoot);
+      target.epoch > computeEpochAtSlot(block.slot) ? beaconBlockRoot : fromHexString(block.targetRoot);
```

Comment:
Instead of converting `block.targetRoot` to Root, can we convert `beaconBlockRoot` to string and compare them as strings below?

Reply:
will do that